### DISCUSSION TITLE
Configuration Help

### DISCUSSION THREAD

**Discussion by paulfears** [MetaMask/snaps#1020] [Q&A]
When I try to import some depenencies I get this error.
"ParseError: 'import' and 'export' may appear only with 'sourceType: module'"
Is there a way to configure the snaps cli to treat all packages like modules?
Or any ideas on how to fix this.

-thanks

**Comment by ritave** [comment]
Are you trying to use `import` and `export` keywords in a JavaScript (.js) snap source file?

If yes, that's not going to work, you have to use `exports.*` way ([CommonJS](https://en.wikipedia.org/wiki/CommonJS)) of exporting your callbacks.
